jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -582,6 +617,21 @@ ecma_utf8_string_to_number (const lit_utf8_byte_t *str_p, /**< utf-8 string */
 
       e_in_lit = e_in_lit * 10 + digit_value;
 
+      /** Sanity check: If the exponent part is bigger than 10^3 we should return with Infinity
+        * because log10(Number.MAX_VALUE) ~= 308.2547.
+        * If the exponent part is less than -10^3  we should return with 0
+        * because log10(Number.MIN_VALUE) ~= -323.3062.
+        * In addition this check prevents e_in_lit overflow even
+        * if the convertable number's exponent part bigger than UINT32_MAX. */
```

Comment:
This is unconventional doc comment style. Please use
```
/**
 * Comment here.
 */
```
format. (Note the position of text and asterisks after the first line.) Look at existing comments as example. (Style change to be applied not only here but to all newly added comments.)